analyzer/runtime/visitors.go

```diff
@@ -121,6 +126,46 @@ func VisitCall(call *sdkTypes.Call, result *sdkTypes.CallResult, handler *CallHa
 				return fmt.Errorf("evm call: %w", err)
 			}
 		}
+	case "rofl.Create":
+		if handler.RoflCreate != nil {
+			var body rofl.Create
+			if err := cbor.Unmarshal(call.Body, &body); err != nil {
+				return fmt.Errorf("unmarshal rofl create: %w", err)
+			}
+			if err := handler.RoflCreate(&body); err != nil {
+				return fmt.Errorf("rofl create: %w", err)
+			}
+		}
+	case "rofl.Update":
+		if handler.RoflUpdate != nil {
+			var body rofl.Update
+			if err := cbor.Unmarshal(call.Body, &body); err != nil {
+				return fmt.Errorf("unmarshal rofl update: %w", err)
+			}
+			if err := handler.RoflUpdate(&body); err != nil {
+				return fmt.Errorf("rofl update: %w", err)
+			}
+		}
```

Comment:
Is there any relevant information in the CallResult for rofl transactions? Or is it left for a future ticket?

Reply:
"rofl.Create" has App ID, others are empty. But we don't have anything to do with it at the moment, will leave it as a followup https://github.com/oasisprotocol/nexus/issues/807